roles/wireguard/templates/peers.j2

```diff
@@ -5,7 +5,7 @@
 {% for peer in wireguard_peers %}
 - WireGuardPeer:
   - FriendlyName: {{ peer.name }}
-  - AllowedIPs: {{ peer.allowed_ip }}/32
+  - AllowedIPs: {{ peer.allowed_ip }}
```

Comment:
In my usecase i am setting always only single IP address as `AllowedIPs`. That's why is the `/32` netmask hardcoded in the template. As a result, i don't need write netmask for every IP in peers configuration in the group vars.

But can see the point that if you want to set a different netmask, you don't have this option here. For backward compatibility could you please change the line to something like this?
```
- AllowedIPs: {{ peer.allowed_ip }}{% if not '/' in peer.allowed_ip %}/32{% endif %}
```

Reply:
Ok I get your point. I have updated it with your suggestion.

Reply:
The tests fail now because the if-statement removes the newline from the AllowIPs line (raw):
"---\n#\n# This is simple jinja2 file used to generate peer variable based on dynamic input.\n#\n  - WireGuardPeer:\n    - FriendlyName: user1\n    - AllowedIPs: 10.213.213.2/32    - PublicKey: \"xxxx=\"\n    - PresharedKey: \"xxxx=\n\"\n    - PersistentKeepalive: 25\n"

Reply:
There is an option to add "#jinja2: trim_blocks:False" at the top of the template. Ref: https://stackoverflow.com/questions/22350175/how-do-i-get-an-ansible-template-to-honor-new-lines-after-a-conditional